### PR TITLE
:bento: Add animated weather icons

### PR DESCRIPTION
**Pull request by frinyvonnick**

### CODE REVIEW

**Review by Taranys — CHANGES_REQUESTED** [review]
Can you add an example of how integrate this into app ?

**Comment by frinyvonnick** [comment]
I made an example in LandingPage

File: src/renderer/components/LandingPage.vue [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <div id="wrapper">
     <main>
       <h1>Hello world!</h1>  
+      <img style="width: 100px;" src="../assets/day.svg" />
```
Comment (Taranys) [inline src/renderer/components/LandingPage.vue:5]: We really have to use an  tag ? No way to use svg directly ?

**Review by Taranys — APPROVED** [review]

File: src/renderer/components/LandingPage.vue (hunk 1 above)
Comment (frinyvonnick) [reply src/renderer/components/LandingPage.vue:5]: Didn't find an easy way to do it... 😢

**Pull request merged**